tests/integration/workflows/test_workflows_tasks_upload.py

```diff
@@ -65,3 +71,72 @@ def test_store_record_does_not_raise_in_the_orcid_receiver(mock_attempt_push, ap
         })
 
         store_record(obj, None)  # Does not raise.
+
+
```

Comment:
we should add a test case when a  with a new ``publisher`` root and an existing ``arxiv``

Reply:
True!